modules/ml/src/test/java/org/apache/ignite/ml/composition/boosting/convergence/ConvergenceCheckerTest.java

```diff
@@ -23,6 +23,7 @@
 import org.apache.ignite.ml.composition.ModelsComposition;
 import org.apache.ignite.ml.composition.boosting.loss.Loss;
 import org.apache.ignite.ml.dataset.impl.local.LocalDatasetBuilder;
+import org.apache.ignite.ml.environment.LearningEnvironmentBuilder;
```

Comment:
Do we need this import statement?

Reply:
Yeah, for `isConverged` method.

Reply:
Not sure, IDEA highlights it as unused import.

Reply:
Fixed, I've looked into `ConvergenceChecker`.